### ISSUE TITLE
[BUG] Crash when adding a badge.

### ISSUE THREAD

**Issue by aminecmi** [Ashok-Varma/BottomNavigation#134]
Hi @Ashok-Varma , 

I'm using the `2.0.1` version, and I'm having the following crash when trying to add a badge.

```
java.lang.IllegalAccessError: Illegal class access: 'ACTIVITY' attempting to access 'com.ashokvarma.bottomnavigation.BadgeItem' (declaration of 'ACTIVITY' appears in /data/app/APK.apk)
```

Here is the code that triggers it (it's Kotlin)
```
val tabNewBadge = TextBadgeItem().setBorderWidth(4)
    .setBackgroundColorResource(R.color.primary)
    .setText("")
val tabNew = BottomNavigationItem(
    R.drawable.ic_fiber_new_black_24dp,
    getString(R.string.tab_new)
).setActiveColor(accent)
  .setBadgeItem(tabNewBadge) // <= this crashes.
```
Is there a way to solve this ?

**Comment by Ashok-Varma** [comment]
@aminecmi i will have a look at it i have no problems in java Code. In kotlin i need to have a look at it

**Comment by Ashok-Varma** [comment]
@aminecmi verified i am on it

**Comment by aminecmi** [comment]
Thanks!

**Comment by Ashok-Varma** [comment]
Fixed in 2.0.2. With in an hour it will be up.

**Comment by aminecmi** [comment]
Can confirm. This fixes the issue.
